### PR TITLE
Jc/add manual

### PR DESCRIPTION
**Pull request by JulienCalbert**
Add first version of the manual.

### CODE REVIEW

File: docs/src/index.md [hunk 1]
```diff
@@ -19,13 +19,13 @@ Rather than relying on closed-form analysis of a model of the dynamical system,
...
 
+
 ## Structure of the documentation 
 
 The documentation is organised as follows.
-
-* The **Examples** section contains a few examples of solving problems with Dionysos. Start with [Getting started](https://dionysos-dev.github.io/Dionysos.jl/dev/generated/Getting%20Started/) if you want to get familiar with Dionysos.
 * The **Manual** section contains all the useful information to use Dionysos as a user.
 * The **API Reference** sections contains all the functions that we currently use in Dionysos. 
+* The **Examples** section contains a few examples of solving problems with Dionysos. Start with [Getting started](https://dionysos-dev.github.io/Dionysos.jl/dev/generated/Getting%20Started/) if you want to get familiar with Dionysos.
```
Comment (adrienbanse) [inline docs/src/index.md:28]: Why don't we use HTML and local paths for images such as 

```Dionysos Logo```

?

File: docs/src/manual/abstraction-based-control.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Abstraction-based control
+
+Given a mathematical description of the system dynamics and the specifications describing the desired closed-loop behavior of the system, 
+**abstraction-based control** techniques involve synthesizing a **correct-by-construction** controller through a **systematic** three-step procedure. 
+First, both the original system and the specifications are transposed into an abstract domain, resulting in an abstract system and corresponding abstract specifications. 
+This step generally involves a complete discretization of the state and input spaces, typically with uniform hyperrectangles.
+Next, an abstract controller is synthesized to solve this abstract control problem. Finally, the third step, referred to as the **refinement procedure**, involves deducing a controller for the original control problem from the abstract controller. The value of this approach lies in the substitution of the original system (often an infinite system) with a finite system, which enables it to leverage powerful control tools in the domain of symbolic control. This three steps procedure is illustrated on the following figure.
+
```
Comment (adrienbanse) [inline docs/src/manual/abstraction-based-control.md:8]: We could put the image here, and not as a link?

Same, we could just put it in the assets and then use 
```Abstraction-based control```

File: docs/src/manual/manual.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+| [`Ellipsoid abstraction`](https://dionysos-dev.github.io/Dionysos.jl/dev/reference/Optim/#Dionysos.Optim.Abstraction.EllipsoidsAbstraction.Optimizer) | Full | Cover | Ellipsoid | Piece-wise affine | Determinisitic | Discrete-time affine | [`State-feedback Abstractions for Optimal Control of Piecewise-affine Systems`](https://arxiv.org/abs/2204.00315) |
+| [`Lazy ellipsoid abstraction`](https://dionysos-dev.github.io/Dionysos.jl/dev/reference/Optim/#Dionysos.Optim.Abstraction.LazyEllipsoidsAbstraction.Optimizer) | Partial | Cover | Ellipsoid | Piece-wise affine | Determinisitic | Discrete-time non-linear | Not yet published |
+
+**Non abstraction-based solver types implemented in Dionysos.jl:**
+
+| Type          | Description | Reference |
```
Comment (adrienbanse) [inline docs/src/manual/manual.md:62]: The descriptions are missing here

File: docs/src/manual/manual.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+| [`Bemporad Morari`](https://dionysos-dev.github.io/Dionysos.jl/dev/reference/Optim/#Dionysos.Optim.BemporadMorari.Optimizer) |  | [`Control of systems integrating logic, dynamics, and constraints`](https://www.sciencedirect.com/science/article/abs/pii/S0005109898001782)
+| [`BranchAndBound`](https://dionysos-dev.github.io/Dionysos.jl/dev/reference/Optim/#Dionysos.Optim.BranchAndBound.Optimizer) |  | [`Abstraction-based branch and bound approach to Q-learning for hybrid optimal control`](https://proceedings.mlr.press/v144/legat21a.html)
+
+
+**Solver interface**
+
+Each solver is defined by a module which must define the structure [`AbstractOptimizer`](https://jump.dev/MathOptInterface.jl/stable/reference/models/#MathOptInterface.AbstractOptimizer) and implement the [`Optimize!`](https://jump.dev/MathOptInterface.jl/stable/reference/models/#MathOptInterface.optimize!) function.
+For example, for the SCOTS solver, this structure and function are defined as follows
+
+```
```
Comment (adrienbanse) [inline docs/src/manual/manual.md:73]: I think if you add ` ```julia `, then you have the julia syntax
Like in this comment: 
```julia
function foo(a)
    return a
end
a = 3
b = foo(a)
```

**Review by adrienbanse — COMMENTED** [review]
Thanks a lot @JulienCalbert, super nice manual

I left a few comments, and there is the problem with the nested list..

File: docs/src/manual/abstraction-based-control.md (hunk 2 above)
Comment (JulienCalbert) [reply docs/src/manual/abstraction-based-control.md:8]: I will try the local reference path

File: docs/src/manual/manual.md (hunk 3 above)
Comment (JulienCalbert) [reply docs/src/manual/manual.md:62]: Yeah I will add a short description of these two solvers.

File: docs/src/manual/manual.md [hunk 5]
```diff
@@ -24,10 +24,7 @@ Description of the core of the Dionysos.jl package, the [`src`](https://github.c
 
 The system types supported in Dionysos.jl are:
 * [`MathematicalSystems`](https://juliareach.github.io/MathematicalSystems.jl/latest/lib/types/#MathematicalSystems.AbstractSystem), which proposes generic and flexible system definitions (e.g.     discrete-time/continuous-time, constrained, noisy systems), such that, for example, the system [`MathematicalSystems.NoisyConstrainedAffineControlDiscreteSystem`](https://juliareach.github.io/MathematicalSystems.jl/latest/lib/types/#MathematicalSystems.NoisyConstrainedAffineControlDiscreteSystem) of the form $$x(k+1) = A x(k) + B u(k) + c + D w(k), \ x(k)\in\mathcal{X}, \ u(k)\in\mathcal{U},\ w(k)\in\mathcal{W}\ \forall k$$
-where 
-    * $\mathcal{X}$ is the state constraints;
-    * $\mathcal{U}$ is the input constraints;
-    * $\mathcal{W}$ is the noise constraints.
+where $\mathcal{X}$ is the state constraint, $\mathcal{U}$ is the input constraint and $\mathcal{W}$ is the noise constraint.
```
Comment (adrienbanse) [inline docs/src/manual/manual.md:27]: 👍

**Review by adrienbanse — APPROVED** [review]
Once the tests are OK we can merge

File: docs/src/manual/manual.md (hunk 5 above)
Comment (JulienCalbert) [reply docs/src/manual/manual.md:27]: To format I used julia-1.8, seems that the problem is julia-1.6

Comment (adrienbanse) [reply docs/src/manual/manual.md:27]: Which is weird because Actions is set up with Julia 1.3.0 😆

**Pull request merged**